src/date-range-picker/calendar/grids/index.tsx

```diff
@@ -114,6 +114,7 @@ export const Grids = ({
     }
 
     switch (e.keyCode) {
+      case KeyCode.space:
```

Comment:
This was previously handled with a separate onKeyDown handler attached to every day button.